#!/usr/bin/env python
# field of view configuration

PACKAGE="cob_3d_mapping_semantics"
#import roslib; roslib.load_manifest(PACKAGE)

from math import pi

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("distance_min", double_t, 3, "minimum distance", 0.5, 0, 3)
gen.add("distance_max", double_t, 3, "maximum distance", 2.5, 0, 3)
gen.add("area_min", double_t, 3, "minimum area", 0.3, 0, 5)
gen.add("area_max", double_t, 3, "maximum area", 3, 0, 10)

exit(gen.generate(PACKAGE, "dynamic_reconfigure_node", "supporting_plane_extraction_node"))
